packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -329,17 +334,18 @@ class _TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
 }
 
 class _Table<T> extends StatefulWidget {
-  const _Table(
-      {Key key,
-      @required this.itemCount,
-      @required this.columns,
-      @required this.columnWidths,
-      @required this.rowBuilder,
-      this.startAtBottom = false})
-      : super(key: key);
+  const _Table({
+    Key key,
+    @required this.itemCount,
+    @required this.columns,
+    @required this.columnWidths,
+    @required this.rowBuilder,
+    this.populateInReverse = false,
```

Comment:
should we just name this `reverse` to match the ListView `reverse` named paramater?

Reply:
Sure; done.